_posts/2021-08-09-status-update.md

```diff
@@ -0,0 +1,72 @@
+
+- [`puppetlabs-docker#764`][puppetlabs-docker-pr-764]: "Remove stderr empty check to avoid docker_params_changed failures when warnings appear", thanks to [cedws][cedws]
+- [`puppetlabs-docker#763`][puppetlabs-docker-pr-763]: "Duplicate declaration statement: docker_params_changed is already declared", thanks to [basti-nis][basti-nis]
+- [`puppetlabs-mysql#1420`][puppetlabs-mysql-pr-1420]: "Set ordering of acceptance tests", thanks to [ghoneycutt][ghoneycutt]
+- [`puppetlabs-stdlib#1196`][puppetlabs-stdlib-pr-1196]: "Fix ensure_packages", thanks to [cocker-cc][cocker-cc]
+- [`facterdb#180`][facterdb-pr-180]: "release 1.8.0", thanks to [bastelfreak][bastelfreak]
+- [`pdk-docker#34`][pdk-docker-pr-34]: "(MAINT) Update bionic image tag to use date-based tag", thanks to [nkanderson][nkanderson]
+- [`rspec-puppet#11`][rspec-puppet-pr-11]: "Add Ruby 3 support", thanks to [bastelfreak][bastelfreak]
+- [`metadata-json-lint#117`][metadata-json-lint-pr-117]: "Update rubocop requirement from ~> 0.50.0 to ~> 0.57.2", thanks to [dependabot[bot]][dependabot[bot]]
+- [`metadata-json-lint#116`][metadata-json-lint-pr-116]: "Add GitHub actions + badges", thanks to [bastelfreak][bastelfreak]
+
+Check [all the ways to reach us]({% link _posts/2021-01-20-reaching-out.md %}) if you want to directly contact us about anything module related.
+
+## New Module / Gem Releases
+
+The following modules were released this week:
+
+- [`puppetlabs-apache`][puppetlabs-apache] (`6.4.0`)
+- [`puppetlabs-postgresql`][puppetlabs-postgresql] (`7.3.0`)
+
+  [puppetlabs-apache]: https://github.com/puppetlabs/puppetlabs-apache
+  [puppetlabs-postgresql]: https://github.com/puppetlabs/puppetlabs-postgresql
+  [puppetlabs-docker-pr-764]: https://github.com/puppetlabs/puppetlabs-docker/pull/764
+  [cedws]: https://github.com/cedws
+  [puppetlabs-docker-pr-763]: https://github.com/puppetlabs/puppetlabs-docker/pull/763
+  [basti-nis]: https://github.com/basti-nis
+  [puppetlabs-mysql-pr-1420]: https://github.com/puppetlabs/puppetlabs-mysql/pull/1420
+  [ghoneycutt]: https://github.com/ghoneycutt
+  [puppetlabs-stdlib-pr-1196]: https://github.com/puppetlabs/puppetlabs-stdlib/pull/1196
+  [cocker-cc]: https://github.com/cocker-cc
+  [facterdb-pr-180]: https://github.com/voxpupuli/facterdb/pull/180
+  [bastelfreak]: https://github.com/bastelfreak
+  [pdk-docker-pr-34]: https://github.com/puppetlabs/pdk-docker/pull/34
+  [nkanderson]: https://github.com/nkanderson
+  [rspec-puppet-pr-11]: https://github.com/puppetlabs/rspec-puppet/pull/11
+  [metadata-json-lint-pr-117]: https://github.com/voxpupuli/metadata-json-lint/pull/117
+  [dependabot[bot]]: https://github.com/apps/dependabot
+  [metadata-json-lint-pr-116]: https://github.com/voxpupuli/metadata-json-lint/pull/116
+
+## Other Work
+
+<!-- check https://tickets.puppetlabs.com/secure/RapidBoard.jspa?rapidView=1176&quickFilter=8745 for other tickets closed out this week that should be mentioned here -->
```

Comment:
```suggestion
We have moved `github-changelog-generator` to  `puppetlabs` namespace .

This will help to add features and bug fixes which will generates the changelog for all our supported modules.
```